### PR TITLE
clarify team admission involves a voting process

### PR DESCRIPTION
**Pull request by vito**
also clarify an instance where conversations may be private, and add a
suggestion for where/how team processes should be documented

Signed-off-by: Alex Suraci <asuraci@vmware.com>

### CODE REVIEW

File: README.md
Comment (aoldershaw) [inline README.md:141]: Slightly confused about this part - what is meant by "determining the required votes" here? Above, it mentions:

> being accepted into a team is entirely subjective and the barrier to entry will vary from team to team

Which I took to mean that if someone wants to join e.g. maintainers, it's the maintainers team that would decide what's required to grant team membership - but this seems counter to my interpretation.

Also, it states that there must be enough approving reviewers to pass the voting process. Does that mean that every request to join a team must get 66% approval from within the team?

Comment (vito) [reply README.md:141]: > Slightly confused about this part - what is meant by "determining the required votes" here? Above, it mentions:

The community team only needs to determine the number of votes required to pass, based on the current team size. They don't determine whether they can join or even know the criteria - that's up to the team members. They just watch for the number of votes and merge when it's met. It could be automated at some point. The destination team won't always be able to merge because not all teams can push to the governance repo.

Would this be sufficient to clarify?

```suggestion
pass the [voting process](#voting). The **community** team is responsible for
determining the number of required votes and merging when they have all been acquired.
```

> Which I took to mean that if someone wants to join e.g. maintainers, it's the maintainers team that would decide what's required to grant team membership - but this seems counter to my interpretation.
> 
> Also, it states that there must be enough approving reviewers to pass the voting process. Does that mean that every request to join a team must get 66% approval from within the team?

Each vote is subjective to the individual team members reviewing, but you would still need the 66% approval required, which is what this attempts to clarify. I guess right now it's easy to confuse the 'barrier to entry' with the voting process, but it's actually talking about the individual votes. [I can fix that.](https://youtu.be/RymXiada3uE?t=16)

(66% is just the default btw, for all voting matters - but teams may choose to define their own threshold or voting process, which it says under the voting section. Technically if a team changes their process the community team would have to somehow know how many votes to expect for future applications though. 🤔)

Happy to field suggestions for alternatives. I figure applying the regular majority vote roles here is maybe a bit heavy handed but probably safer than a 1 or 2 approvers vote where a single org could just vote themselves to a majority. That's a bit of a doomsday scenario though and hopefully unrealistic.

Comment (aoldershaw) [reply README.md:141]: Your clarifications make sense, thanks.

> I figure applying the regular majority vote roles here is maybe a bit heavy handed but probably safer than a 1 or 2 approvers vote where a single org could just vote themselves to a majority

Yeah, that's kind of what I was thinking. On one hand, it makes sense to have 66% agreement, but on the other, it feels potentially like a big barrier to entry, especially if team members aren't necessarily responsive.

Btw, is there a process in mind for *removing* members from a team? e.g. they aren't acting in good faith, or just stopped responding and are hindering the voting process. Presumably that'd also just follow a regular 66% (by default) vote?

Comment (vito) [reply README.md:141]: > Yeah, that's kind of what I was thinking. On one hand, it makes sense to have 66% agreement, but on the other, it feels potentially like a big barrier to entry, especially if team members aren't necessarily responsive.

Happy to change it, but unresponsive members would be a problem with anything else that requires a vote too; if a team ends up in that situation they might want to just adjust their voting requirements.

> Btw, is there a process in mind for _removing_ members from a team? e.g. they aren't acting in good faith, or just stopped responding and are hindering the voting process. Presumably that'd also just follow a regular 66% (by default) vote?

Haha hadn't thought about that. I suppose people should be able to just remove themselves regardless, and for removing other people I guess the regular voting rules would make sense. Guess I should write that up too.

Comment (vito) [reply README.md]: Pushed a new revision that I think addresses all this - lmk what you think!

**Review by aoldershaw — APPROVED** [review]
LGTM! Thanks for the clarifications!

**Pull request merged**
